src/auth/LoginForm.tsx

```diff
@@ -187,6 +199,10 @@ const LoginForm = ({
         <JSForm {...formProps} _internalId={_internalId} />
       </div>
     );
+  } else if (apexForm) {
+    // If logged in and have finished onboarding questions for an apex form, then we need to redirect back to application
+    window.location.href = 'http://localhost:3000';
```

Comment:
should this be set in the dashboard or via URL param when redirected to the apex form?